scripts/helpers.js

```diff
@@ -1,16 +1,39 @@
+var moment = require('moment');
 var urllib = require('url');
 
 var utils = require('../lib/utils');
-
 var pkg = require('../package');
+
+var isUrl = utils.isUrl;
+var urljoin = utils.urljoin;
+
 var aframeCurrentSha = 'master';
 try {
   aframeCurrentSha = pkg.dependencies.aframe.split('#')[1];
-} catch (e) {
-}
+} catch (e) {}
 
-var isUrl = utils.isUrl;
-var urljoin = utils.urljoin;
+hexo.extend.helper.register('blog_attribution', function (author) {
+  // Twitter handle.
+  if (author[0] === '@') {
+    return '<a class="blog-attribution" href="https://twitter.com/' + author + '">' +
+      author + '</a>';
+  }
+  // Make a space imply that it is not a Github username. Return as is.
+  if (author.indexOf(' ') !== -1) {
```

Comment:
or I had a different idea: perhaps instead of having `@cvanw` denote Twitter handles, perhaps allow `twitter:cvanw` vs. `github:cvan`?